goodmap/db/local_json_db.py

```diff
@@ -12,11 +12,15 @@ def __init__(self, config):
         self.data_file_path = config["data_file_path"]
 
     def get_data(self):
-        with open(self.data_file_path, 'r') as file:
-            return json.load(file)
+        try:
+            with open(self.data_file_path, "r") as file:
+                return json.load(file)
+        except FileNotFoundError:
+            with open("example_data.json", "r") as file:
+                return json.load(file)
```

Comment:
No. Example data is not part of 'production'. Let it not be part of business logic. Give some meaningfull error here. "Please provide file with json data" or sth similar.